implants/golem/Cargo.toml

```diff
@@ -18,6 +18,13 @@ structopt = { workspace = true }
 itertools = { workspace = true }
 thiserror = { workspace = true }
 lsp-types = { workspace = true }
+aes-gcm = { version = "0.10.3", features = ["aes"] }
```

Comment:
Move these the workspace cargo.toml and use the `dep-name = { workspace = true }`